dateutil/test/test_isoparser.py

```diff
@@ -322,14 +323,14 @@ def test_parse_tzstr_fails(tzstr, exception):
 def __make_date_examples():
     dates_no_day = [
         date(1999, 12, 1),
-        date(2016, 2, 1),
+        date(2016, 2, 1)
     ]
 
     if six.PY3:
         # strftime does not support dates before 1900 in Python 2
         dates_no_day.append(date(1000, 11, 1))
 
-    date_no_day_fmts = ('%Y%m', '%Y-%m')
+    date_no_day_fmts = ['%Y-%m']
```

Comment:
Why the change from tuple to list? I'm not married to using a tuple, just curious. (Though also this might be a situation where we can drop this parameter entirely, I'd have to read the tests more thoroughly to see).

Reply:
In the ```__make_data_examples()``` function ```o = it.product(dates_no_day, date_no_day_fmts)``` is used to create the examples, and ```date_no_day_fmts``` is a tuple with two formats we want the date to be in. However now we need only one format so a list with a singular element is used. This is similar to line 383.

Reply:
@kirit93 I don't understand why this would make you switch to a list, though.

```python
>>>import itertools as it
>>>from datetime import date
>>>dates_no_day = [date(1999, 12, 1), date(2016, 2, 1)]
>>>dates_no_date_fmts = ('%Y-%m',)
>>>o = it.product(dates_no_day, dates_no_date_fmts)
>>>print(list(o))
[(datetime.date(1999, 12, 1), '%Y-%m'), (datetime.date(2016, 2, 1), '%Y-%m')]

```

Reply:
You're right, I could have left it as a tuple. I just changed it to be a list because that's how it was done in the test on line 383, no other reason.